src/components/sections/social-media-links.svelte

```diff
@@ -0,0 +1,137 @@
+<script>
+</script>
+
+<section class="social-media-links-section">
+  <a
+    href=" https://www.instagram.com/acmcsuf/"
+    class="social-link"
+    target="”_blank”"
```

Comment:
Here, remove the replace the line `target="”_blank”"` with `target="_blank"`.

Reply:
The typo exists in some other lines in this file as well, so be sure to replace the typo where it is also needed.